Client/README.md

```diff
@@ -694,42 +694,184 @@ mine-sensor,id=132-987-655,customer="California Miner",hostname=example.com,sens
 
 ### Handle the Events
 
-#### Handle the Success write
+Events that can be handle by WriteAPI EventHandler are:
+- `WriteSuccessEvent` - for success response from server
+- `WriteErrorEvent` - for unhandled exception from server
+- `WriteRetriableErrorEvent` - for retriable error from server
+- `WriteRuntimeExceptionEvent` - for runtime exception in background batch processing
+
+Number of events depends on number of data point to collect in batch Batch option in WriteAPI (default setting is 1000) - in case 
+of one data point in batch option, event is handled for each point, independently on used writing method (even for mass writing of data like 
+`WriteMeasurements`, `WritePoints` and `WriteRecords`). 
```

Comment:
```suggestion
Number of events depends on number of data points to collect in batch. The batch size is configured by `BatchSize` option (default size is `1000`) - in case 
of one data point, event is handled for each point, independently on used writing method (even for mass writing of data like 
`WriteMeasurements`, `WritePoints` and `WriteRecords`). 
```